packages/pnpm/package.json

```diff
@@ -30,14 +30,17 @@
     "@pnpm/outdated": "1.0.8",
     "@pnpm/package-store": "1.2.0",
     "@pnpm/server": "1.1.1",
+    "@pnpm/store-controller-types": "1.1.0",
     "@pnpm/store-path": "1.0.4",
     "@pnpm/types": "2.0.0",
     "@pnpm/utils": "0.9.0",
+    "@types/archy": "0.0.31",
```

Comment:
Had to add archy to `pnpm` to display the results in a tree-like structure. Is this the right place to add it?

Reply:
yes